File: pecahan.cpp
// #include <iostream>
#include<bits/stdc++.h>
using namespace std;
int main() {
    unsigned int  A, B, C, D;
    cin >> A >> B >> C >> D;

    

    if (A*D>B*C) {
        cout << "lebih besar" << endl;
    } else if (A*D<B*C) {
        cout << "lebih kecil" << endl;
    } else {
        cout << "sama" << endl;
    }

    return 0;
}

File: pecahan.py
# Membaca input
A, B = map(int, input().split())
C, D = map(int, input().split())

# Menghitung perkalian silang
nilai_A = A * D
nilai_C = C * B

# Membandingkan pecahan
if nilai_A == nilai_C:
    print("sama")
elif nilai_A > nilai_C:
    print("lebih besar")
else:
    print("lebih kecil")
